### DISCUSSION TITLE
Is it possible to add a one-time buyout payment to the paid model (Perpetual license)

### DISCUSSION THREAD

**Discussion by 307836997** [rustdesk/rustdesk#5530] [Feature Request]
RT

**Comment by rustdesk** [comment]
Of course, we just do not know how to define the price. sales@rustdesk.com

**Reply by Enlargee** [reply]
If it's of any help - for home users, supporting their friends/families a relatively low price would be sensible, and a great incentive for all of us to get the paid version. If not overpriced, I would happily buy a perpetual rustdesk for home/non-commercial use only in a heartbeat.

**Reply by dinger1986** [reply]
Problem is how do you "prove" you are only supporting friends and family and not using it commercially?

**Reply by 307836997** [reply]
我认为可以可以给个人/家庭 永久许可一些限制,例如可注册数量,或者同时使用数量.

**Reply by 307836997** [reply]
40美元左右?是否合适,可以限制注册数量或者同时使用数量,防止商业用途.

**Reply by Enlargee** [reply]
Perhaps it's not a matter of having to prove it. There are many companies that provide free for personal use versions, and those are explicitly EULAd as such. There's no doubt a huge number of companies using pirated software as well, but those that do are subject to closer scrutiny and auditing, particularly the larger companies where rustdesk pro would be most often used.

By making RD accessibly priced for home users, sure you're gonna get dishonest folks using that for business purposes but you're also going to have captured a vast market share because as it stands right now RD, is unquestionably superior to teamviewer in performance.  At a glance, a 50 connection addressbook, or 1-3 group limit would probably not affect the vast majority of homelab/self-host users.

Resilio as an example, is completely suitable for home PC based file syncing and though there's some neat and fancy features on the pro version, I can get by without ever missing those because they're just not relevant on a home setup.
